evalml/automl/engine/sequential_engine.py

```diff
@@ -0,0 +1,26 @@
+from evalml.automl.engine import EngineBase
+
+
+class SequentialEngine(EngineBase):
+    """The default engine for the AutoML search. Trains and scores pipelines locally, one after another."""
+
+    def evaluate_batch(self, pipelines):
+        """Evaluate a batch of pipelines using the current dataset and AutoML state.
+
+        Arguments:
+            pipelines (list(PipelineBase)): A batch of pipelines to be fitted and evaluated.
+
+        Returns:
+            list (int): a list of the new pipeline IDs which were created by the AutoML search.
+        """
+        if self.X_train is None or self.y_train is None:
```

Comment:
If we're checking that `self.X_train, self.y_train` aren't None, should we also do this for the AutoMLSearch object? If that's None, this call should also fail
![image](https://user-images.githubusercontent.com/22552445/108222530-e2939f80-7106-11eb-8550-9dddbf5c3808.png)

Reply:
That's true. I suppose it can't hurt to add that. Since there's only one call site at the moment, and if we didn't set automl search there all the tests will fail, I will hold off on adding that right now.